x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/enrich/20_standard_index.yml

```diff
@@ -145,6 +145,7 @@ enrich documents over _bulk via an alias:
           - '{"baz": "lazy", "c": 2}'
           - '{"index": {"_id": "3"}}'
           - '{"baz": "slow", "c": 3}'
+  - match: { errors: false }
```

Comment:
Fun story, this test failed because it relies on this bulk operation to create the failure store. I lost about an hour debugging why it wasn't only to find out that the bulk operation was returning failures due to the failure store not initializing. Checking this here identifies that the bulk failed instead of just the next command in the test "failing".